src/ProjectTemplates/test/Infrastructure/TemplateTests.props.in

```diff
@@ -23,6 +23,20 @@
       RuntimePackRuntimeIdentifiers="${SupportedRuntimeIdentifiers}" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Workaround when there is no vNext SDK available, copy known apphost/framework reference info from 5.0 -->
+    <KnownAppHostPack
+      Include="@(KnownAppHostPack->WithMetadataValue('TargetFramework', 'netcoreapp5.0'))"
+      TargetFramework="${KnownAppHostPackOrFrameworkReferenceTfm}"
```

Comment:
@wtgodbe any idea why this isn't defined?

Reply:
Not sure, but we can just use `DefaultNetCoreTargetFramework`